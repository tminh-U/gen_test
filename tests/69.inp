12
1 3 14 10 16 19 13 7 8 3 3 9 
10
11 12
1 6
3 4
2 7
10 11
9 10
7 7
1 10
1 7
3 10
