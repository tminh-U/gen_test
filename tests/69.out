4
48
3
28
4
3
2
72
56
18
